exporter/datadogexporter/README.md

```diff
@@ -8,54 +8,4 @@
 | Supported pipeline types | traces, metrics, logs|
 | Distributions            | [contrib], [AWS] |
 
-
-This exporter sends metric, traces and logs to [Datadog](https://datadoghq.com). For environment specific setup instructions visit the [Datadog Documentation](https://docs.datadoghq.com/tracing/setup_overview/open_standards/otel_collector_datadog_exporter/).
-
-> Please review the Collector's [security
-> documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security.md),
-> which contains recommendations on securing sensitive information such as the
-> API key required by this exporter.
```

Comment:
Let's keep this, it was asked for by AWS when the exporter was added to the AWS distro.